lib/views/pages/home/content/ranking/ranking_card.dart

```diff
@@ -0,0 +1,133 @@
+import "package:flutter/material.dart";
+import "package:proxima/models/ui/ranking/ranking_element_details.dart";
+import "package:proxima/models/ui/user_avatar_details.dart";
+import "package:proxima/views/components/content/publication_header/user_profile_pop_up.dart";
+import "package:proxima/views/components/content/user_avatar/user_avatar.dart";
+
+/// A class that displays a ranking card.
```

Comment:
a ~class~, a widget

Reply:
Same for other documentations in this PR (please apply everywhere).

Reply:
You are right !

Fixed in: [b58916a](https://github.com/ProximaEPFL/proxima/pull/322/commits/b58916ab148ea2e6c1db72272e13d2538c2aecf9)

Thank you !